plugins/catalog-import/src/components/ImportComponentPage.tsx

```diff
@@ -37,6 +40,28 @@ export type ConfigSpec = {
   config: PartialEntity[];
 };
 
+function manifestGenerationAvailable(configApi: ConfigApi): boolean {
+  return configApi.has('integrations.github');
+}
+
+function repositories(configApi: ConfigApi): string[] {
+  const integrations = configApi.getConfig('integrations');
```

Comment:
This should all ideally have used the new `@backstage/integrations` package, but it's not very conveniently exposed on the frontend yet. So OK for now!